frontend/src/lib/components/LemonCalendar/LemonCalendarRange.tsx

```diff
@@ -0,0 +1,146 @@
+import { LemonCalendar, LemonCalendarProps } from 'lib/components/LemonCalendar/LemonCalendar'
+import React, { useEffect, useState } from 'react'
+import { dayjs } from 'lib/dayjs'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconClose } from 'lib/components/icons'
+import { formatDateRange } from 'lib/utils'
+
+export interface LemonCalendarRangeProps {
+    value?: (string | null)[]
+    onChange: (date: string[]) => void
```

Comment:
Since it's a range, it should always have `[start, end]`, so this type could be a tuple `[string, string]` for stronger typing

Reply:
Makes sense. I tried that earlier, but got several errors and bailed, probably from code that's now deleted. Fixed.